Pod/Classes/URBNDataSourceAdapter.h

```diff
@@ -9,16 +9,18 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
+NS_ASSUME_NONNULL_BEGIN
+
 typedef NS_ENUM(NSUInteger, URBNSupplementaryViewType) {
     URBNSupplementaryViewTypeHeader,
     URBNSupplementaryViewTypeFooter
 };
 
-typedef NSString *(^URBNSupplementaryViewReuseIdentifierBlock) (NSString *kind, NSIndexPath *indexPath);
-typedef NSString *(^URBNReuseableIdentifierBlock) (id item, NSIndexPath *indexPath);
+typedef NSString * _Nonnull (^URBNSupplementaryViewReuseIdentifierBlock) (NSString *kind, NSIndexPath * _Nullable indexPath);
```

Comment:
Is there a reason you're using `_Nonnull` and `_Nullable` when you use `nullable` below?